plugins/chat/lib/tasks/chat_doc.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+task "chat:doc" do
+  ["plugins/chat/assets/javascripts/discourse/services/chat-api.js"].each do |file|
+    origin = File.join(Rails.root, file)
+    filename = File.basename(file, ".*")
+    destination = File.join(Rails.root, "plugins/chat/docs/#{filename}.md")
+    config = File.join(Rails.root, ".jsdoc")
+
+    `jsdoc2md -c #{config} #{origin} > #{destination}`
```

Comment:
I expect this is still using the globally-installed version. `yarn jsdoc2md ...` should do it